nservicebus/azure-storage-queues/performance-tuning.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Azure Storage Transport Performance Tuning
+summary: Performance Tuning Azure Storage as transport
+reviewed: 2016-04-21
+tags:
+- Azure
+- Cloud
+- Persistence
+- Transport
+---
+
+include: azure-storage-performance-tuning
+
+
+## Parallel Message Retrieval
+
+In NServiceBus Azure Storage Queue Transport Version 7 and higher, multiple parallel read operations are used to improve message throughput. The amount of parallel read operations is the square root of the configured [message processing concurrency](/nservicebus/operations/tuning.md). You can increase or decrease this value if needed by using the `DegreeOfReceiveParallelism` configuration parameter. See [Azure Storage Queues Transport Configuration](/nservicebus/azure-storage-queues/configuration.md) on how to use this parameter.
```

Comment:
> remove the personal language of "you"

Pro-tip - if you use MarkdownPad with hack, then it automatically detects "forbidden" words and doesn't compile.